### PR TITLE
Advance swift exercise

### PR DESCRIPTION
**Pull request by Kruna1Pate1**
### Exercises for advance swift

- Collection types
- Functions
- Enum, Class, Struct
- Higher Order Functions
- Types in Initialization and deinitialization
- Overrides
- Reference Counting
- Interchange Dictionary keys & values
- Optional chaining
- Inheritance
- Nested type
- Error Handing
- Converting errors to optional value
- Disabling error propagation
- Type casting and checking
- Extension
- Optional chaining with nil-coalescing
- Strong, Weak & Unowned reference with example
- Reference cycle with example
- Breaking Reference cycle
- Overview of Access modifier
- Basics of Protocols
- Access control example
- Protocol and Delegate
- Optional protocol requirement

### CODE REVIEW

File: Demo/MyPlayground.playground/Pages/AdvanceExercise.xcplaygroundpage/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,1180 @@
...
+var grade1 = 7.0
+var grade2 = 9.0
+var grade3 = 5.0
+var yourGrade = 8.0
+let avg = (grade1 + grade2 + grade3 + yourGrade) / 4
+print(yourGrade > avg ? "above average" : "below average")
+
+
+// 5. Tipping
+var mealCost:Double = 3.5
```
Comment (purva-SS) [inline Demo/MyPlayground.playground/Pages/AdvanceExercise.xcplaygroundpage/Contents.swift]: add one space after colon

File: Demo/MyPlayground.playground/Pages/AdvanceExercise.xcplaygroundpage/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,1180 @@
...
+let tip:Int = 20
```
Comment (purva-SS) [inline Demo/MyPlayground.playground/Pages/AdvanceExercise.xcplaygroundpage/Contents.swift]: add one space after colon

**Comment by Kruna1Pate1** [comment]
Added space after colon

**Review by purva-SS — CHANGES_REQUESTED** [review]
Screenshot 2023-03-27 at 12 11 52 PM
Screenshot 2023-03-27 at 12 12 09 PM
Remove unnecessary folder and comments

**Comment by Kruna1Pate1** [comment]
I can't find folder named `ChangeMainScreen`. I think it's old folder because content of that folder was deleted in this PR [Combine separate projects](https://github.com/Kruna1Pate1/simform-ios/pull/1). And empty folders aren't tracked by git.

**Pull request merged**